analysis_options.yaml

```diff
@@ -1,8 +1,5 @@
 # Specify analysis options.
 #
-# Until there are meta linter rules, each desired lint must be explicitly enabled.
-# See: https://github.com/dart-lang/linter/issues/288
```

Comment:
I agree that the link is not relevant, but isn't it still true that we have to manually enable every lint?

It'd be great if we had a test that checked that when Dart is rolled, the lints in this file (including those commented out) exactly match the list of lints in the version of Dart...

Reply:
Right, filed https://github.com/flutter/flutter/issues/101596 for that.